api/http/handler/edgestacks/edgestack_create.go

```diff
@@ -22,6 +22,46 @@ func (handler *Handler) edgeStackCreate(w http.ResponseWriter, r *http.Request)
 		return &httperror.HandlerError{http.StatusBadRequest, "Invalid query parameter: method", err}
 	}
 
+	edgeStack, createSwarmStackError := handler.createSwarmStack(method, w, r)
```

Comment:
The error handling here is a bit weird, I'd rather keep the usual `edgeStack, err` error handling and return the following:

```go
		return &httperror.HandlerError{http.StatusInternalServerError, "Unable to create Edge stack", err}
```

I don't think that you need to pass the httpResponseWriter `w` parameter to this function if it's only used for the error handling. 

See my comment below about error handling.